addons/l10n_ec_website_sale/tests/test_l10n_ec_website_sale.py

```diff
@@ -1,16 +1,21 @@
 from odoo.tests.common import HttpCase, tagged
-from odoo.addons.l10n_ec_edi.tests.common import TestEcEdiCommon
+
+from odoo.addons.account.tests.common import AccountTestInvoicingCommon
 
 
 @tagged('post_install_l10n', 'post_install', '-at_install')
-class TestUi(HttpCase, TestEcEdiCommon):
+class TestUi(HttpCase, AccountTestInvoicingCommon):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
 
     def test_checkout_address_ec(self):
+        self.env.company.country_id = self.env.ref('base.ec').id,
```

Comment:
You'll have a tuple here
```suggestion
        self.env.company.country_id = self.env.ref('base.ec').id
```